docs/Reference/Configuration-File.md

```diff
@@ -71,6 +83,50 @@ Details of the operator's engine component.
 
 Address (host and port) of the engine.
 
+### `ha`
+
+Configure ConsenSys Rollups to be [highly available](../Concepts/High-Availability.md).
```

Comment:
Is this section of the config file just empty? What's the purpose of it?

Reply:
This section has been created so that HA can have its own section. There are no options for it.
[ha.kafka] is a child of [ha].
It's similar to [this example](https://github.com/toml-lang/toml#example).